src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -16,44 +16,82 @@ public class FindMyRecyclingController {
     @Autowired
     IFacilityService facilityService;
 
+    /**
+     * index page
+     * @return
+     */
```

Comment:
JavaDoc should be descriptive of what the method is doing,  over and above what one can determine from the method name itself.